docs/Versioning.md

```diff
@@ -44,6 +45,7 @@ A variety of tools can be used to create ONNX models. Unless otherwise noted, pl
 
 |Tool|Recommended Version|Supported ONNX version(s)|
 |---|---|---|
+| 0.5.0 |[Latest stable](https://pytorch.org/get-started/locally/)|1.2-1.5*<br>*may require [ONNX version converter](https://github.com/onnx/onnx/blob/master/docs/VersionConverter.md) to convert to desired opset #*|
```

Comment:
This line seems to have an issue. (I think it shouldn't be generated). The changes were automated with the script - tools\python\update_version.py. Will investigate.

Reply:
The logic in update_version.py needed refining (I refined it). Shouldn't generate this line next time.